.github/workflows/main.yml

```diff
@@ -45,13 +45,7 @@ jobs:
       - name: Run cppcheck
         shell: bash
         run: |
-          cppcheck --error-exitcode=1 -j2 -DRANGECHECK -D_WIN32 -Isrc src toolsrc 2> stderr.txt
-          RET=$?
-          if [ -s stderr.txt ]
-          then
-            cat stderr.txt
-          fi
-          exit $RET
+          cppcheck --error-exitcode=1 -q -U_WIN32 -Isrc opl setup textscreen
```

Comment:
What's `-q`?

Reply:
`-q, --quiet          Do not show progress reports.`
It print only errors.

Reply:
Thanks! That wasn't in the PDF manual that I downloaded. 🤷

Reply:
`man cppcheck`  is more useful than that PDF manual.

Reply:
Apparently. 😁